apps/lend/src/components/PageMarketList/index.tsx

```diff
@@ -13,19 +13,20 @@ import MarketListNoResult from '@/components/PageMarketList/components/MarketLis
 import MarketListItemContent from '@/components/PageMarketList/components/MarketListItemContent'
 import TableSettings from '@/components/PageMarketList/components/TableSettings/TableSettings'
 import usePageVisibleInterval from '@/ui/hooks/usePageVisibleInterval'
+import { useOneWayMarketMapping } from '@/entities/chain'
 
 const MarketList = (pageProps: PageMarketList) => {
   const { rChainId, isLoaded, searchParams, api, updatePath } = pageProps
 
   const activeKey = _getActiveKey(rChainId, searchParams)
+  const {data: marketMapping} = useOneWayMarketMapping(rChainId)
```

Comment:
```suggestion
  const { data: marketMapping } = useOneWayMarketMapping(rChainId)
```
What's wrong with prettier?

Reply:
seems it's not part of the pipeline

Reply:
If I run `npm run format` I get 244 files changed